helpers/spark-submit.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import pwd
+import subprocess
+import logging
+import argparse
+
+USER_HOME_DIR_ENT_IDX = 5
+EXIT_CODE_BAD_KUBECONFIG = -100
+EXIT_CODE_BAD_CONF_ARG = -200
+
+def expand_arg(arg: str) -> str:
+    if os.environ.get(arg):
+        return os.environ[arg]
+    else:
+        return arg
+
+if __name__ == "__main__":
+    logging.basicConfig(format='%(asctime)s %(levelname)s %(message)s', level=logging.INFO)
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--master", default=None, type=str)
```

Comment:
Could you just add some help descriptions here to the arguments. That also looks very nice if you render --help